### ISSUE TITLE
Version 2.2.0: “Unexpected EOF” connecting to SQL Server

### ISSUE THREAD

**Issue by fabriba** [pymssql/pymssql#703]
Hi there,
I get this every time I try to connect to database since updating to v.2.2.0 about 5hrs ago. old version still works well.
Unfortunately I don't have many details about the ms-servers I connect to, other than I believe they are on version 12.0.6433.1  and 11.0.7507.2 (this is listed in the connection header, not sure if accurate, not much of a mssql admin person myself)

**Comment by termim** [comment]
Do they require SSL? What is your client OS? Could you provide output of
`python -c "import pymssql; print(pymssql.version_info())"`

**Comment by fredrik-corneliusson** [comment]
I have the same issue.
This is the pymssql version_info:
```System:
platform.system()       : Linux
platform.architecture() : ('64bit', 'ELF')
platform.libc_ver()     : ('glibc', '2.28')
Python:
sys.version             : 3.8.8 (default, Feb 19 2021, 17:55:44)
[GCC 8.3.0]
pymssql:
VERSION            : (2, 2, 0)
version            : 2.2.0
full_version       : 2.2.0
dbversion          : freetds v1.2.18
_mssql:
VERSION            : (2, 2, 0)
version            : 2.2.0
full_version       : 2.2.0
dbversion          : freetds v1.2.18
login_timeout      : 60
min_error_severity : 6
```

**Comment by termim** [comment]
Could you set environment `TDSDUMP=/tmp/freetds.log`, run your failing script again and show the log file here?

**Comment by termim** [comment]
Also could you try to install pymssql from here: [test.pypi.org](https://test.pypi.org/project/pymssql/2.2.1.dev2/). It could be glibc/openssl version related.

**Comment by fredrik-corneliusson** [comment]
Tested "2.2.1.dev2" and the issue is gone.
Thanks for the quick action!

**Comment by termim** [comment]
It should be fixed in [2.2.1](https://pypi.org/project/pymssql/2.2.1/). Thanks for the report!
